OpenInfraSummit2023/Open-Collaboration-viewed-through-the-eyes-of-a-PO.md

```diff
@@ -0,0 +1,35 @@
+# Open Collaboration and Development in SCS - viewed through the eyes of a Product Owner
+
+Track: Open Development
+
+Authors: Felix Kronlage-Dammers
+
+Technical Level: Beginner
+
+## Abstract (1000 chars)
+
+The Sovereign Cloud Stack (SCS) orchestrates various standardization and development
+efforts in an open and collobaritve fashion. The projects development efforts typically 
+happen in the corresponding upstream project - done through funding and tender projects
+just as much as through interested parties at the various cloud service providers that
+utilize the SCS stack. Coordination happens in recurring meetings facilitated by the
+product owners at the SCS project - embracing the four opens!
+In the course of this talk the Product Owner for the IaaS Reference Implementation of the
+Sovereign Cloud Stack, which is based on OpenStack obviously, will dive into how the
+development process works and will share insights on what went well in the first years
+and where things needed (and still need) improvement and how these hurdles are tackled.
```

Comment:
I suggest to split this rather long sentence.

> In the course of this talk the Product Owner for the IaaS Reference Implementation of the
Sovereign Cloud Stack, which is based on OpenStack obviously, will dive into how the
development process is managed. Felix will share insights on what went well in the first years, where things needed (and still need) improvement and how these hurdles are tackled.